ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.test.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import configureMockStore from 'redux-mock-store';
+import sinon from 'sinon';
+import { renderWithProvider } from '../../../../../test/lib/render-helpers';
+import EditGasToolTip from './edit-gas-tooltip';
+
+const LOW_GAS_OPTION = {
+  gasLimit: 21000,
+  maxFeePerGas: '2.010203381',
+  maxPriorityFeePerGas: '1.20004164',
+  maxFeePerGasValue: '2.383812808',
+  maxPriorityFeePerGasValue: '1.5',
+  origin: 'metamask',
+};
+
+const MEDIUM_GAS_OPTION = {
+  gasLimit: 21000,
+  maxFeePerGas: '2.383812808',
+  maxPriorityFeePerGas: '1.5',
+  maxFeePerGasValue: '2.383812808',
+  maxPriorityFeePerGasValue: '1.5',
+  origin: 'metamask',
+};
+
+const HIGH_GAS_OPTION = {
+  gasLimit: 21000,
+  maxFeePerGas: '2.920638342',
+  maxPriorityFeePerGas: '2',
+  maxFeePerGasValue: '2.383812808',
+  maxPriorityFeePerGasValue: '1.5',
+  origin: 'metamask',
+};
+
+const renderComponent = (props) => {
+  const mockStore = {
+    metamask: {
+      provider: {
+        type: 'test',
+      },
+    },
+  };
+
+  const store = configureMockStore()(mockStore);
+
+  return renderWithProvider(
+    <EditGasToolTip {...props} t={sinon.stub()} />,
```

Comment:
Do you need to pass `t` into the component? If you `import { renderWithProvider } from '../../../../../test/jest'` rather than `import { renderWithProvider } from '../../../../../test/lib/render-helpers'` I think `t` is already stubbed so you shouldn't have to worry about this.

Reply:
this is required because of that bug I'm dealing with the translationContext not working in this component, I've had to make the component expect a method prop called `t`.